specifications/passive/android_google_api-1.0.0.yml

```diff
@@ -0,0 +1,14 @@
+#======================================== Google APIs =============================================#
```

Comment:
Please rename model to `Activity` and rename file to `google_activity-1.0.0.yml`.